_posts/2024-11-26-OSPREY-Wins-HPCwire-Award.md

```diff
@@ -0,0 +1,63 @@
+> – Valerie Hayot-Sasson, a postdoctoral scholar at the University of Chicago and ANL.
+
+OSPREY aims to enhance pandemic response by enabling epidemiological researchers and health officials to utilize HPC resources and facilitates data-driven decision-making to make better and timely public health decisions. Researchers from  Argonne National Lab (ANL)  and the University of Chicago used Globus, <a href="https://emews.org" target="_blank">EMEWS</a> (Extreme-scale Model Exploration with Swift), and the newly developed <a href="https://github.com/NSF-RESUME/aero" target="_blank">AERO</a> (Automated Event-based Research Orchestration) platform to integrate automated workflows, data curation, and epidemiological analyses to facilitate rapid collaboration and development during health crises.
+
+> Our focus with OSPREY has been to develop broadly applicable capabilities to improve decision-making for epidemic response, tying together advanced data, modeling, computing, and automation approaches. This recognition by the HPCwire editors and thought leaders in HPC is both gratifying and helps us further disseminate our open science platform.
+> – Jonathan Ozik, a principal computational scientist at ANL and senior scientist at the consortium for Advanced Science and Engineering with Public Health Science affiliation at the University of Chicago.
+
+<a href="https://www.globus.org/news/globus-receives-multiple-honors-in-hpcwire-annual-awards" target="_blank">Read Globus’ press release for more info about this award</a>.
+
+<hr>
+
+## The Creation of OSPREY
+
+COVID-19 had an unprecedented impact on scientific collaboration. The pandemic and its broad response from the scientific community forged new relationships among domain experts, mathematical modelers, and scientific computing specialists. Computationally, however, it revealed critical gaps in researchers’ ability to exploit advanced computing systems. These challenges include accessing scalable computing systems, porting models and workflows to new systems, sharing data of varying sizes, and producing results that others can reproduce and validate.
+
+To address these challenges, a multi-institutional team from ANL, UChicago, and Inria centre at Université Côte d’Azur Sophia Antipolis, came together to draw on their experiences in supporting public health decision-makers during the COVID-19 pandemic and developed OSPREY (Open Science Platform for Robust Epidemic analYsis). OSPREY seeks to lower the barriers to and automate epidemiological model analyses, monitoring, and rapid response on HPC resources. The team published a paper about their work prototyping the open science platform in May 2023, <a href="https://ieeexplore.ieee.org/abstract/document/10196539?casa_token=JYWij7orZnMAAAAA:IWXYvrY9HAAzg4fcgNOwyRhLmFQDbZ2MF2m4iRgBj-hvrc2LMaP0ZWrBZOX_ZXcPkA0Bo0xu-wNP" target="_blank">read it here</a>.
```

Comment:
a better URL for the paper is just "https://doi.org/10.1109/IPDPSW59300.2023.00143"